common/kvstore/src/test/java/org/apache/spark/util/kvstore/InMemoryStoreSuite.java

```diff
@@ -86,6 +88,52 @@ public void testMultipleObjectWriteReadDelete() throws Exception {
     }
   }
 
+  @Test
+  public void testMultipleTypesWriteReadDelete() throws Exception {
```

Comment:
This test was only available for LevelDBSuite so I copied here to test new API addition.